### PR TITLE
(WIP) WL-981 edxapp raw queries

### PR DESCRIPTION
**Pull request by afzaledx**

### CODE REVIEW

File: edx_salesforce/management/commands/sync_salesforce.py [hunk 1]
```diff
@@ -34,8 +34,43 @@ def handle(self, *args, **options):
...
         with connections['default'].cursor() as cursor:
-            # cursor.execute()
+            query= "SELECT auth_user.id, auth_user.username, auth_user.email FROM edxapp.auth_user, edxapp.student_userattribute WHERE auth_user.id=student_userattribute.user_id AND student_userattribute.name='created_on_site' AND student_userattribute.value='{domain}';".format(
+                domain=site_domain
+            )
+            cursor.execute(query)
+            auth_users = cursor.fetchall()
+
+            for auth_user in auth_users:
+                users_with_attributes[auth_user[0]] = {
```
Comment (afzaledx) [inline edx_salesforce/management/commands/sync_salesforce.py]: Perhaps we can do it in all in this loop if we have multiple cursors.

**Comment by afzaledx** [comment]
sample data output: 
```
        {
            8: {
                'auth_user.email': 'afzal+1@edx.org',
                'auth_user.username': 'afzal',
                'auth_userprofile.name': 'Afzal Wali',
                'auth_userprofile.country': '',
                'auth_userprofile.year_of_birth': 1976
                'auth_userprofile.language': '',
                'student_userattribute.utm_source': 'utm_source_v',
                'student_userattribute.utm_medium': 'utm_medium_v',
                'student_userattribute.utm_campaign': 'utm_campaign_v',
                'student_userattribute.utm_term': 'utm_term_v',
                'student_userattribute.utm_content': 'utm_content_v',
            }
        }
```

@douglashall

**Comment by afzaledx** [comment]
The above queries are 'working', but with my limited raw sql knowledge, I was not able to optimize them. I will work on optimization tomorrow.

File: edx_salesforce/management/commands/sync_salesforce.py [hunk 2]
```diff
@@ -34,8 +34,54 @@ def handle(self, *args, **options):
...
+            auth_users = cursor.fetchall()
+
+            for auth_user in auth_users:
+                users_with_attrs[auth_user[0]] = {
+                    'auth_user.username': auth_user[1],
+                    'auth_user.email': auth_user[2],
+                }
+                user_ids.append(auth_user[0])
+
+            for user_id in users_with_attrs:
```
Comment (afzaledx) [inline edx_salesforce/management/commands/sync_salesforce.py]: Perhaps we can do it in all in the loop above if we have multiple cursors.

**Comment by douglashall** [comment]
Closing this as I have manually incorporated the work into https://github.com/edx/edx-salesforce/pull/3.

**Pull request closed without merging**